docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG PYTORCH="1.6.0"
+ARG CUDA="10.1"
+ARG CUDNN="7"
+
+FROM pytorch/pytorch:${PYTORCH}-cuda${CUDA}-cudnn${CUDNN}-devel
+
+ENV TORCH_CUDA_ARCH_LIST="6.0 6.1 7.0+PTX"
+ENV TORCH_NVCC_FLAGS="-Xfatbin -compress-all"
+ENV CMAKE_PREFIX_PATH="(dirname(which conda))/../"
+
+RUN apt-get update && apt-get install -y ffmpeg libsm6 libxext6 git ninja-build libglib2.0-0 libsm6 libxrender-dev libxext6 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install MMCV
+RUN pip install mmcv-full==1.3.8 -f https://download.openmmlab.com/mmcv/dist/cu101/torch1.6.0/index.html
+
+# Install MMClassification
+RUN conda clean --all
+RUN git clone https://github.com/open-mmlab/mmclassification.git
```

Comment:
Should we use an absolute path? 
Refers to https://github.com/open-mmlab/mmaction2/blob/aa26715d5c6a0991af4ead5b5ea03c46dd65dcee/docker/Dockerfile#L20-L21.